File: nomor1.cpp
#include <iostream>

using namespace std;

int main(){
    int N;
    cout << "Nilai N = ";
    cin >> N;

    int counter = 0;
    int number = 1;

    while(counter < N){
        if(number % 3 == 0 && number % 7 == 0){
            if(counter != N - 1){
                cout << "N, ";
            }else{
                cout << "N" ;
            }
            counter ++;
        }else if(number % 3 == 0 || number % 7 == 0){
            if(counter != N - 1){
                cout << number <<", ";
            }else{
                cout << number ;
            }
            counter ++;
        }
        number++;
    }


    return 0;
}

File: nomor1.py
N = int(input("Nilai N = "))

counter = 0         #counter untuk banyak angka yang akan ditampilkan
number = 3          #Angka yang akan diiterasi secara berurut

while(counter < N):                         #Melakukan Loop sebanyak counter
    if (number % 3 == 0 and number % 7 == 0):
        if(counter != N-1):
            print("N, ", end="")
        else :
            print("N", end="")
        counter +=1
    elif(number % 3 == 0 or number % 7 == 0):
        if(counter != N-1):
            print(number,', ', end="")
        else:
            print(number, end="")
        counter +=1
    number +=1
